master/reference/kube-controllers/configuration.md

```diff
@@ -55,3 +55,42 @@ The following environment variables can be used to configure the {{site.prodname
 
 [in-cluster-config]: https://kubernetes.io/docs/tasks/access-application-cluster/access-cluster/#accessing-the-api-from-a-pod
 [kubeconfig]: https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/
+
+## About each controller
+
+### Node controller
+
+The node controller automatically cleans up configuration for nodes that no longer exist.
+
+The node controller is not enabled by default. However, the Calico self-hosted manifests do enable this controller.
+
+To enable the node controller, perform the following two steps.
+
+1. Add "node" to the list of enabled controllers in the environment for kube-controllers. For example: `ENABLED_CONTROLLERS=workloadendpoint,profile,policy,node`
+1. Configure calico/node with a Kubernetes node reference by adding the following snippet to the environment section of the calico-node daemon set.
```

Comment:
`calico/node` --> `{{site.nodecontainer}}`
the other one can be `calico-node` --> `{{site.noderunning}}`